avocado/plugins/replay.py

```diff
@@ -213,11 +213,12 @@ def run(self, args):
                                         for _ in args.multiplex_files]
             else:
                 mux = jobdata.retrieve_mux(resultsdir)
-                if mux is None:
-                    log.error('Source job multiplex data not found. Aborting.')
+                if mux is None:     # Fallback to multiplex_files
```

Comment:
I'm a bit lost here.  Is this comment a "TODO" and you intended to write a fallback mechanism?  I can't really see it falling back to `multiplex_files` here.